### ISSUE TITLE
[Rails 5] Fix spec error - missing text "Keep private for"

### ISSUE THREAD

**Issue by lizconlan** [mysociety/alaveteli#5116]
```
viewing requests in alaveteli_pro a pro user viewing one of their own requests the request is not embargoed shows the option to add an embargo
Failure/Error: expect(page).to have_content "Keep private for"
  expected #has_content?("Keep private for") to return true, got false
# ./spec/integration/alaveteli_pro/view_request_spec.rb:48:in `block (5 levels) in <top (required)>'
# ./spec/integration/alaveteli_dsl.rb:106:in `block (2 levels) in using_pro_session'
# /home/vagrant/bundle/ruby/2.3.0/gems/capybara-3.5.1/lib/capybara.rb:347:in `using_session'
# /home/vagrant/bundle/ruby/2.3.0/gems/capybara-3.5.1/lib/capybara/dsl.rb:22:in `using_session'
# ./spec/integration/alaveteli_dsl.rb:105:in `block in using_pro_session'
# ./gems/alaveteli_features/lib/alaveteli_features/spec_helpers.rb:7:in `with_feature_enabled'
# ./spec/integration/alaveteli_dsl.rb:104:in `using_pro_session'
# ./spec/integration/alaveteli_pro/view_request_spec.rb:46:in `block (4 levels) in <top (required)>'
```

Required by #3969

**Comment by lizconlan** [comment]
Aha, possibly a parser error...

Rails 4 (excerpt):
```
Privacy This request is public Change privacy Keep private for: Choose a duration
```

Rails 5 (excerpt):
```
Privacy
This request is public
Change privacy
\" class=\"embargo__submit js-embargo-submit\">
```

**Comment by lizconlan** [comment]
oh - _sigh_ filename error

**Comment by lizconlan** [comment]
Oh no - I've swapped one error for another. Not sure whether Rails or Capybara...

`it 'allows the user to add an annotation'` (in `shared_examples_for_viewing_requests`)

Rails 4 (incorrectly) renders the `views/comment/_submit.html.erb` partial and outputs "Post annotation"

Rails 5 renders `views/alaveteli_pro/comment/_submit.html.erb` and outputs "Add annotation"

```
1) viewing requests in alaveteli_pro a pro user viewing one of their own requests allows the user to add an annotation
     Failure/Error: click_button("Post annotation")

     Capybara::ElementNotFound:
       Unable to find button "Post annotation"
     Shared Example Group: "allows annotations"
       called from ./spec/integration/alaveteli_pro/view_request_spec.rb:20
```

**Comment by lizconlan** [comment]
(It's Rails, will probably do a lazy fix here)

**Comment by lizconlan** [comment]
Closed via #5122
